axelrod/tests/unit/test_moran.py

```diff
@@ -330,6 +331,20 @@ def test_iter(self):
         mp = MoranProcess((p1, p2))
         self.assertEqual(mp.__iter__(), mp)
 
+    def test_population_plot(self):
+        # Test that can plot on a given matplotlib axes
+        axelrod.seed(15)
+        N = 5
+        players = [random.choice(axelrod.basic_strategies)() for _ in range(N)]
```

Comment:
Same comment about the `basic_strategies` would this potentially change the axes in lines 345, 346 if and when new strategies come in?

For the purpose of the test let's just use `demo_strategies` (or any other hard coded list: perhaps just a cooperator and a defector?)